src/shared/reducer.ts

```diff
@@ -33,3 +35,41 @@ export const reducer = combineReducers({
     activeMusic: ephemeralMusicReducer,
   }),
 });
+
+const specialCharacterReducer = createReducer(initialSyncedState, (builder) => {
+  builder.addCase(initiativeTrackerSetCurrentEntry, (state, action) => {
+    if (!state.initiativeTracker.currentEntryId) return;
+    const newInitiativeEntry =
+      state.initiativeTracker.entries.entities[
+        state.initiativeTracker.currentEntryId
+      ];
+    // should have already been updated by the call to `intermediateState = combinedReducer(state, action)` at the bottom
+    assert(state.initiativeTracker.currentEntryId === action.payload);
+    if (!newInitiativeEntry || newInitiativeEntry.type === "lairAction") return;
+    newInitiativeEntry.characterIds.map((id) => {
+      const characterEntry = state.characters.entities[id];
+      if (characterEntry?.currentlyConcentratingOn) {
+        characterEntry.currentlyConcentratingOn.roundsLeft--;
+      }
+    });
+  });
+});
+
+function crossSliceReducer(state: SyncedState, action: AnyAction) {
+  switch (action.type) {
+    case initiativeTrackerSetCurrentEntry.toString(): {
+      return {
+        ...state,
+        characters: specialCharacterReducer(state, action).characters,
+      };
+    }
+    default:
+      return state;
+  }
+}
+
+export function reducer(state: SyncedState, action: AnyAction): SyncedState {
```

Comment:
```suggestion
export function reducer(state: SyncedState = initialSyncedState, action: AnyAction): SyncedState {
```

I think you need to add the default/initial state to the reducer, which might also fix the type error.